ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/PicoCliDownstreamTlsTrustOptions.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.config;
+
+import tech.pegasys.ethsigner.core.config.DownstreamTrustOptions;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import picocli.CommandLine.Option;
+
+public class PicoCliDownstreamTlsTrustOptions implements DownstreamTrustOptions {
+  @Option(
+      names = "--downstream-http-tls-disallow-ca-signed",
+      description = "Disable CA signed server certificates to be trusted.",
+      arity = "0")
+  private boolean tlsDisallowCaSignedCert = false;
```

Comment:
Can we do something odd here - make the variable a positive statement "useSystemCA = true"; (but keep a negative cmdline) -- then we don't need the negation in the isCaSignedServerCertificateAllowed.... maybe this is better :/ Ask someone else :)

Reply:
I tried it (even with fallbackValues) ... however, PicoCLI likes to use "false" when you don't specify boolean values with arity 0.